app/oxalis/security/Environment.scala

```diff
@@ -12,19 +12,25 @@ import net.ceedubs.ficus.readers.ArbitraryTypeReader._
 
 import scala.concurrent.ExecutionContext
 import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.duration._
 import scala.concurrent.duration.FiniteDuration
+import org.joda.time.DateTime
 
 
-class EnvironmentOxalis @Inject()(configuration: Configuration)(implicit val executionContext: ExecutionContext) extends Environment[User ,CookieAuthenticator]{
+class EnvironmentOxalis @Inject()(configuration: Configuration)(implicit val executionContext: ExecutionContext) extends Environment[User ,CombinedAuthenticator]{
+  val eventBusObject = EventBus()
+  val cookieSettings = configuration.underlying.as[CookieAuthenticatorSettings]("silhouette.cookieAuthenticator")
+  //val tokenSettings = BearerTokenAuthenticatorSettings("X-Auth-Token", None, 365 days) //maybe change the authenticatorExpiry and write it in a .conf file
```

Comment:
you did that, i think? then we can remove this line

Reply:
Yes. I forgot to delete the comment